docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -7,6 +7,15 @@ The ClickHouse connector allows querying tables in an external
 query data in the databases on that instance, or combine it with other data
 from different catalogs accessing ClickHouse or any other supported data source.
 
+Requirements
+------------
+
+To connect to a ClickHouse instance, you need:
+
+* ClickHouse version 21.6 or higher.
```

Comment:
We test with 20.8. See https://github.com/trinodb/trino/blob/master/plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java#L29

Reply:
Got it, thanks. Updated.